client/app/telecom/orders/telecom-orders.html

```diff
@@ -10,78 +10,35 @@ <h1 data-translate="telecom_orders_follow_up_title"></h1>
-                                </i>
-                            </td>
-                        </tr>
-                        <tr data-ng-show="OrdersCtrl.loading">
-                            <td colspan="5">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                        <tr data-ng-if="OrdersCtrl.ordersDetails.length == 0 && !OrdersCtrl.loading">
-                            <td colspan="5"
-                                class="text-center"
-                                data-translate="telecom_orders_follow_up_no_orders">
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-            <div class="mt-2"
-                 data-ng-show="OrdersCtrl.orders.length > OrdersCtrl.perPage"
-                 data-pagination-front
-                 data-go-to-page="false"
-                 data-items="OrdersCtrl.orders"
-                 data-paginated-items="OrdersCtrl.ordersDetails"
-                 data-current-page="OrdersCtrl.currentPage"
-                 data-items-per-page="OrdersCtrl.perPage"
-                 data-nb-pages="OrdersCtrl.nbPages"
-                 data-on-transform-item-notify="OrdersCtrl.onTransformItemNotify(item)"
-                 data-page-placeholder="{{currentPage}} / {{nbPages}}"
-                 data-item-per-page-placeholder="{{'telecom_orders_follow_up_items_per_view' | translate}}">
-            </div>
+
+            <oui-datagrid data-rows="OrdersCtrl.ordersDetails">
+                <oui-column title="'telecom_orders_follow_up_header_service' | translate" data-property="name">
+                    <a data-ui-sref="telecom.orders.show({serviceName: $row.xdsl })"
+                        data-ng-bind="$row.name | limitTo: 25">
+                    </a>
+                </oui-column>
+                <oui-column title="'telecom_orders_follow_up_header_type' | translate" data-property="accessType">
+                    <span data-ng-bind="'telecom_orders_follow_up_type_' + $row.accessType | translate">
+                    </span>
+                </oui-column>
+                <oui-column title="'telecom_orders_follow_up_header_status' | translate" data-property="lineStatus">
+                    <span data-ng-bind="'status_' + $row.lineStatus | translate">
+                    </span>
+                </oui-column>
+                <oui-column title="'telecom_orders_follow_up_header_step' | translate" data-property="lastTask.name">
+                    <span data-ng-if="$row.lastTask"
+                            data-ng-bind="'telecom_common_pack_xdsl_order_step_name_' + $row.lastTask.name | translate">
+                    </span>
+                </oui-column>
+                <oui-column title="'telecom_orders_follow_up_header_state' | translate" data-property="lastTask.status" data-sortable>
```

Comment:
Since it would be easy, I think 120 char length are easier to read, but I really don't mind if it stays that way